source/includes/api/_before_you_get_started.md

```diff
@@ -1,27 +1,60 @@
 
-This section covers a list of topics you should be familiar with before using the API. These concepts span the whole API.
+This section covers a list of topics you should be familiar with before using the API. The concepts described in this section span across multiple API endpoints and are fundamental for a better understanding of how to interact with the RW API.
 
 ## Applications
 
-TODO
+As you might have come across while reading these docs, different applications and websites rely on the RW API as the principal source for their data. While navigating through the catalog of available datasets, you will find some datasets used by the [Resource Watch website](https://resourcewatch.org/), others used by [Global Forest Watch](https://www.globalforestwatch.org/). In many cases, applications even share the same datasets!
 
+To ensure the correct separation of content across the different applications that use the RW API, you will come across a field named `application` in many of the API's resources (such as datasets, layers, widgets, and others). Using this field, the RW API allows users to namespace every single resource, so that it's associated only with the applications that use it.
 
-## Authentication
+### Existing applications
 
-TODO
+Currently, the following applications are using the API as the principal source for their data:
 
+* the [Resource Watch website](https://resourcewatch.org/), where the `application` field takes the value `rw`;
+* the [Global Forest Watch website](https://www.globalforestwatch.org/), where the `application` field takes the value `gfw`;
+* the [Partnership for Resilience and Preparedness website](https://prepdata.org/), where the `application` field takes the value `prep`;
+* the [Forest Atlases websites](https://www.wri.org/our-work/project/forest-atlases) for different countries of the world also rely on the RW API - in the case of these websites, the `application` field takes the value `forest-atlas`;
 
-## Roles
+If you would like to see your application added to the list of applications supported by the RW API, please contact us.
+
+### Filtering content by the application field
+
+> Fetching datasets for the Resource Watch application
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?application=rw
+```
+
+> Fetching datasets for the Global Forest Watch application
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?application=gfw
+```
+
+You can use the `application` field as a query parameter filter in many of the resources of the RW API to filter content specific to the given application. Check out on the side some examples of the usage of the `application` field when fetching RW API datasets.
+
+Please keep in mind that some API resources might not use the `application` field. Filtering capabilities may also vary from one resource to another. Always make sure you check the specific documentation for the `application` field of the corresponding API resource.
```

Comment:
It would be good if we could include some "best practices" on how to use the `application` field correctly, since compliance is completely voluntary.

Reply:
(i.e. not enforced by anything)

Reply:
Merged this section (_Usage of applications for access management_) with the section above (_Filtering content by the application field_), into a single section called _Best practices for the application field_, which consolidates the best practices that should be taken into account when using the application field. 

Should be clearer now what is the "default" way of using this field correctly :+1: